charmhelpers/contrib/charmsupport/nrpe.py

```diff
@@ -509,6 +510,37 @@ def add_haproxy_checks(nrpe, unit_name):
         check_cmd='check_haproxy_queue_depth.sh')
 
 
+def add_openvswitch_checks(nrpe, unit_name):
+    """
+    Add checks for openvswitch
+
+    :param NRPE nrpe: NRPE object to add check to
+    :param str unit_name: Unit name to use in check description
+    """
+    enable_sudo_for_openvswitch_checks()
+    nrpe.add_check(
+        shortname='openvswitch',
+        description='Check Open vSwitch {%s}' % unit_name,
+        check_cmd='check_openvswitch.py')
```

Comment:
How about make the name more explicit, e.g. check_ovs_interfaces.py or check_ovs_ifaces.py?

Reply:
I was definitely feeling this would be the start of something that could be expanded to more checks as we identify them.  The interface errors are just MVP for the current need.